src/main/resources/application.properties

```diff
@@ -3,4 +3,6 @@ spring.datasource.url=jdbc:postgresql://localhost/internship-management-system
 spring.datasource.platform=postgresql
 spring.datasource.username=postgres
 spring.datasource.password=123456
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL82Dialect
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL82Dialect
+spring.security.user.name=admin
+spring.security.user.password=admin
```

Comment:
Şimdilik kalsın çalışmanızı engellemesin ama bunların kaldırılması için talep açılsın @yagizharman

Reply:
hocam talep açılsın derken issue olarak mı açayım nasıl yani